superset-frontend/plugins/plugin-chart-table/src/DataTable/hooks/useSticky.tsx

```diff
@@ -246,23 +246,6 @@ function StickyWrap({
       </colgroup>
     );
 
-    // header columns do not have vertical scroll bars,
-    // so we add scroll bar size to the last column
-    const headerColgroup =
-      sticky.hasVerticalScroll && scrollBarSize ? (
-        <colgroup>
-          {colWidths.map((x, i) => (
-            // eslint-disable-next-line react/no-array-index-key
-            <col
-              key={i}
-              width={x + (i === colWidths.length - 1 ? scrollBarSize : 0)}
```

Comment:
@reesercollins just to be triple sure, do you have evidence that this isn't required any longer?

Reply:
I haven't been able to find any instances where this removal causes any breaking changes, however, I am not the most knowledgeable when it comes to the usage of this plugin within superset since our organization uses a custom table viz.